plugins/progress/README.md

```diff
@@ -0,0 +1,47 @@
+### Using the Progress Bar Script
+
+#### Description
+
+This script provides a simple progress bar GUI for the terminal platform. It allows you to visualize the progress of a task being executed in your shell script.
+
+#### Usage
+
+1. **Copy Functions:**
+
+   - Copy the `delay` and `progress` functions directly into your shell script.
```

Comment:
I doubt this is the intended usage, but maybe I need to check the plugin file later.

Reply:
Ok

Reply:
- First, the plugins in OMB are intended for interactive uses. For example, the users' shell functions defined in `.bashrc` can utilize the shell functions defined in the plugins in OMB. However, they are not primarily designed to be used within a standalone shell program, though it might still work.
- Next, the intended way is to specify the plugin name in the array `plugins` in `~/.bashrc` and let `oh-my-bash.sh` load them. Even if a user wants to use the plugin in a standalone shell program, the plugin file is supposed to be sourced using the `source` builtin. Its contents are not supposed to be directly copied into the main file of the shell program.

The "Usage" section of the suggested change should explain the standard usage by the interactive shell functions through the `plugins` array.

If you are sure that one can also use the `progress` plugin in a separate shell program, that usage should be introduced as an extra usage and should instruct users to use `source` instead of copying the contents.